### PR TITLE
PHPUnit dependency

### PR DESCRIPTION
**Pull request by umpirsky**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
         "paragonie/random_compat": "~1.0|~2.0"
     },
     "require-dev": {
+        "phpunit/phpunit": "~5.4",
```
Comment (xabbuh) [inline composer.json]: This is an issue as PHPUnit 5.4 requires PHP 5.6 while this bundle supports PHP 5.3.

Comment (Spomky) [inline composer.json]: `"phpunit/phpunit": "^4.0|^5.0",`

**Comment by scheb** [comment]
Added PHPUnit as a dependency. I'll stick to PHPUnit <= 5.3 for now, because of deprecations warnings starting with PHPUnit 5.4. We cannot update safely to that version yet, because the bundle supports PHP 5.4, which is only supported by PHPUnit 4.

**Comment by xabbuh** [comment]
@scheb Always using the mock builder instead of `createMock()`/`getMock()` can be a solution since it's working with all PHPUnit versions.

**Pull request closed without merging**